reana_cluster/cli/cluster.py

```diff
@@ -249,7 +249,7 @@ def cli_verify_components(ctx):
     '--component',
     default=None,
     help='Specify for which component you want the status'
-         'e.g. job-controller.')
+         'e.g. server.')
```

Comment:
Here too, let's use `workflow-controller` as the example component everywhere.

Please also modify `reana/cli.py` documentation:

```console
reana-dev --help | grep job-con               
          $ cd reana-job-controller
          $ reana-dev git-checkout -b reana-job-controller 72
          $ reana-dev kubectl-delete-pod -c reana-job-controller
```

r-j-c should be docker-build-able, but not kubectl-delete-pod-able.